README.md

```diff
@@ -282,6 +282,10 @@ different Miri binaries, and as such worth documenting:
   that the compiled `rlib`s are compatible with Miri.
   When set while running `cargo-miri`, it indicates that we are part of a sysroot
   build (for which some crates need special treatment).
+* `MIRI_CALLED_FROM_RUSTDOC` when set to any value tells `cargo-miri` that it is
+  running as a child process of `rustdoc`, which invokes it twice for each doc-test
+  and requires special treatment, most notably a check-only build before validation.
```

Comment:
Here "validation" is again; could you replace this by something more specific? I don't know what validation you mean here.

"interpretation" or "running the interpreter" would work, I think.